### DISCUSSION TITLE
Pain meter

### DISCUSSION THREAD

**Discussion by Ryotaiku** [littlesera/LSCG#391] [Ideas]
It'd be cool if pain actions like whipping or shocks caused a build-up like the arousal meter. Maybe when it's full it causes a pass-out like the chloroform or choking.

**Comment by jimmyG1** [comment]
this seems fun

**Comment by jimmyG1** [comment]
thow it might be smart to have a maskist mode

**Comment by littlesera** [comment]
I know a couple of people who would enjoy this >.>

**Reply by anchillabc** [reply]
Why did I see this so late?
